packages/react-examples/src/react-label/Label/Label.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+
+export const LabelExample = () => (
```

Comment:
You can do this in a future PR, but you'll probably want to add `'react-label'` to the list of packages that use `withFluentProvider` in packages/react-examples/.storybook. That'll provide theming for the examples.